bazel-java-deps.bzl

```diff
@@ -86,20 +86,20 @@ def install_java_deps():
             # This effectively means all io.grpc:*, io.netty:*, and `com.google.protobuf:protobuf-java
             # need to be updated with careful consideration.
             # grpc
-            "io.grpc:grpc-api:1.29.0",
-            "io.grpc:grpc-core:1.29.0",
-            "io.grpc:grpc-netty:1.29.0",
-            "io.grpc:grpc-protobuf:1.29.0",
-            "io.grpc:grpc-services:1.29.0",
-            "io.grpc:grpc-stub:1.29.0",
+            "io.grpc:grpc-api:1.35.0",
+            "io.grpc:grpc-core:1.35.0",
+            "io.grpc:grpc-netty:1.35.0",
+            "io.grpc:grpc-protobuf:1.35.0",
+            "io.grpc:grpc-services:1.35.0",
+            "io.grpc:grpc-stub:1.35.0",
             # netty
-            "io.netty:netty-codec-http2:4.1.48.Final",
-            "io.netty:netty-handler:4.1.48.Final",
-            "io.netty:netty-handler-proxy:4.1.48.Final",
-            "io.netty:netty-resolver:4.1.48.Final",
-            "io.netty:netty-tcnative-boringssl-static:2.0.30.Final",
+            "io.netty:netty-codec-http2:4.1.58.Final",
+            "io.netty:netty-handler:4.1.58.Final",
+            "io.netty:netty-handler-proxy:4.1.58.Final",
+            "io.netty:netty-resolver:4.1.58.Final",
+            "io.netty:netty-tcnative-boringssl-static:2.0.36.Final",
             # protobuf
-            "com.google.protobuf:protobuf-java:3.11.0",
+            "com.google.protobuf:protobuf-java:3.14.0",
```

Comment:
Does it make sense to define a few constants to change these in a single place?

Reply:
good idea 84a8a9c287048eeab450143e8d3d0d7170ab5a76